rzk/src/Rzk/TypeCheck.hs

```diff
@@ -2936,7 +2936,7 @@ renderCube
   -> (String -> Maybe RenderObjectData)
   -> String
 renderCube camera rotY renderDataOf' = unlines $ filter (not . null)
-  [ "<svg class=\"zoom\" style=\"float: right\" viewBox=\"-175 -200 350 375\" width=\"150\" height=\"150\">"
+  [ "<svg class=\"zoom tope\" viewBox=\"-175 -200 350 375\" width=\"150\" height=\"150\">"
```

Comment:
Why not one class? E.g.
```suggestion
  [ "<svg class=\"rzk-svg-render\" viewBox=\"-175 -200 350 375\" width=\"150\" height=\"150\">"
```

Reply:
I considered the possibility of the `zoom` class being applied to other elements that might benefit from the zooming behavior, but I guess it doesn't make a difference in this case. I'll consolidate them.